duecredit/injections/mod_scipy.py

```diff
@@ -25,4 +25,130 @@ def inject(injector):
       year =      {2001--},
       url = "http://www.scipy.org/",
       note = {[Online; accessed 2015-07-13]}
-    }"""), use="Scientific tools library")
\ No newline at end of file
+    }"""), description="Scientific tools library")
+
+    # scipy.cluster.hierarchy general references
+    # TODO: we should allow to pass a list of entries
+    injector.add('scipy.cluster.hierarchy', None, BibTeX("""
+    @article{johnson1967hierarchical,
+        title={Hierarchical clustering schemes},
+        author={Johnson, Stephen C},
+        journal={Psychometrika},
+        volume={32},
+        number={3},
+        pages={241--254},
+        year={1967},
+        publisher={Springer}
+    }"""), min_version='v0.4.3',
+                 description="Global references for the module")
```

Comment:
See -- that is how "description" is probably misleading. Before it was "use", with original intent to signal to describe how this functionality is usually used.  E.g. here I probably would have said

```description="Hierarchical clustering", tags=["edu"]```